client/__tests__/e2e/test_persistence_config.yaml

```diff
@@ -0,0 +1,24 @@
+server:
+  app:
+    force_https: true
+    port: 5005
+    debug: false
+    verbose: false
+    disable_auth: false
+
+  multi_dataset:
+    dataroots:
+      test_dataroot:
+        base_url: w
```

Comment:
Persistence tests route via base_url=w with disable_auth=false, which exercises user-scoped saves and reloads.